dali/python/nvidia/dali/external_source.py

```diff
@@ -446,11 +555,15 @@ def __call__(self, *, source = None, cycle = None, name = None, layout = None, c
         else:
             if name is not None:
                 kwargs["name"] = name
+            if no_copy is not None:
+                kwargs["no_copy"] = no_copy
             op_instance = _OperatorInstance([], self, **kwargs)
             op_instance._callback = callback
             op_instance._output_index = None
-            op_instance._group = _ExternalSourceGroup(callback, False, [op_instance], cuda_stream=cuda_stream,
-                                                      use_copy_kernel=use_copy_kernel, batch=batch)
+            op_instance._group = _ExternalSourceGroup(
+                callback, False, [op_instance],
+                cuda_stream=cuda_stream, use_copy_kernel=use_copy_kernel, batch=batch,
+                parallel=parallel, prefetch_queue_depth=prefetch_queue_depth)
```

Comment:
This is what I'm talking about in the comment above.

Reply:
Nice idea.